archinstall/__init__.py

```diff
@@ -33,6 +33,8 @@ def initialize_arguments():
 	parser.add_argument("--creds", nargs="?", help="JSON credentials configuration file")
 	parser.add_argument("--silent", action="store_true",
 						help="Warning!!! No prompts, ignored if config is not passed")
+	parser.add_argument("--dry-run", action="store_true",
+						help="Warning!!! Only creates config, does not install")
```

Comment:
I don't like the Warning!!! in this message. Can we change the help text to "Generates a configuration file and then exits instead of performing an installation"?

Reply:
Sure, sounds way better